cmd/snap/cmd_info.go

```diff
@@ -401,6 +401,14 @@ func (iw *infoWriter) printSummary() {
 	wrapFlow(iw, quotedIfNeeded(iw.theSnap.Summary), "summary:\t", iw.termWidth)
 }
 
+func (iw *infoWriter) maybePrintStoreURL() {
+	if iw.remoteSnap == nil || iw.remoteSnap.StoreURL == "" {
```

Comment:
because you now have it in `aux`, it could be there even with a nil remote

Reply:
Tweaked this a little and left a note. Otherwise we wouldn't see the store url for snaps that don't get refreshed too often until the snapd bits land.